packages/components/src/Breadcrumbs/Breadcrumbs.component.js

```diff
@@ -71,6 +74,7 @@ function Breadcrumbs(props) {
 						bsStyle="link"
 						role="link"
 						title={title || text}
+						aria-label={title}
```

Comment:
Why adding an aria-label if a label and a title are already provided? 
Which information is added for screen readers?

Reply:
I assume that 
* `text` is the label, so the content of the button
* `title` the tooltip

The problem is that the tooltip is more accurate, providing more info than the content.
But `title` is less important than `content` in a11y text order. So screen readers will take the content over the title, so the less accurate/precise text.

So when you have `title` and `content`, we add an `aria-label` with the `title` value to be read by SR over the content.

Reply:
Ok got it thx